portfolio/src/main/webapp/index.html

```diff
@@ -110,11 +110,11 @@ <h2>Got something to say?</h2>
           <form id="comments-form" action="/data" method="POST">
             <div class="form-element">
               <label>Nickname</label>
-              <input class="comment-io" name="nickname" type="text" placeholder="Sam S" pattern="\w+">
+              <input class="comment-io" name="nickname" type="text" placeholder="Sam S" pattern=".+">
```

Comment:
The way this is defined now, it will accept exactly any non-empty input.
A simpler way to do this may be by using the [`required` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required).
Beware: I haven't tried that, so please excuse me if these are doing different things.

Reply:
I actually used the required attr on the description, but purposefully left it off of the nickname line to allow for anonimity. On the backend, a blank nickname resolves to "anon".

Reply:
See PR #28 for more information on that.

Reply:
I see. If the empty string is allowed, then you can just drop the `pattern` attribute (and not use a required attribute). As a reference, the regex `".+"` means [one or more occurrence](https://javascript.info/regexp-quantifiers#shorthands) of any character (so it won't match the empty string).